frontend/src/pages/devices/List.tsx

```diff
@@ -2,16 +2,15 @@ import React from 'react';
 import { Badge, Button, OverlayTrigger, Tooltip } from 'react-bootstrap';
 import { Loading } from '../utils/Loading';
 import { StandardFetch } from '../utils/FetchHelper';
-import { useHistory } from 'react-router';
 import { formatDistance, isAfter, isBefore, sub } from 'date-fns';
+import {Link} from 'react-router-dom';
```

Comment:
Please add spaces around `Link` to make it beautiful. You can include that change in a future PR.